### PR TITLE
enabled auto commit

### PR DESCRIPTION
**Pull request by Pranav-b-7**
https://devopsmx.atlassian.net/browse/OP-19253

### CODE REVIEW

File: scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py [hunk 1]
```diff
@@ -1638,4 +1638,5 @@ def verifyDBVersion(version):
         update_db("4.0.2")      # Note: version here should be updated for each ISD release
     elif migrate_data_flag == 'true':        
         perform_migration("4.0.2")       # pass the ISD version we are performing data migration for (Note: version here should be updated for each ISD release)
+        audit_conn.autocommit = True
```
Comment (rakeshkrmeena) [inline scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py:1641]: @Pranav-b-7 why just audit_conn; why not other connections as well which are used in the perform_migration method (I guess, platform_conn, spindb_orca and spindb_front50) ?

File: scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py [hunk 2]
```diff
@@ -131,6 +98,39 @@ def update_db(version):     # pre-upgrade DB Update
     finally:
         close_connections()
 
+
+def migrate_audit_data():
```
Comment (rakeshkrmeena) [inline scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py:102]: function name like update_audit_db will look more appropriate !

File: scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py (hunk 1 above)
Comment (Pranav-b-7) [reply scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py:1641]: spindb_orca and spindb_front50 are spinnaker DBs . So we need not worry about that.  platform. i missed it. will add them

File: scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py (hunk 2 above)
Comment (Pranav-b-7) [reply scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py:102]: ok sure will do it

**Review by rakeshkrmeena — APPROVED** [review]
review comments are addressed and approved

**Pull request merged**